lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -402,7 +402,7 @@ defmodule RealtimeWeb.RealtimeChannel do
   def limit_channels(%{assigns: %{tenant: tenant, limits: limits}, transport_pid: pid}) do
     key = Tenants.channels_per_client_key(tenant)
 
-    if Registry.count_match(Realtime.Registry, key, pid) > limits.max_channels_per_client do
+    if Registry.count_match(Realtime.Registry, key, pid) > Application.get_env(:realtime, :max_channels_per_client, 100) do
```

Comment:
```suggestion
    if Registry.count_match(Realtime.Registry, key, pid) > limits.max_channels_per_client do
```

this needs to stay as is to respect per tenant limits